src/AutoRest.CSharp/MgmtTest/Generation/MgmtBaseTestWriter.cs

```diff
@@ -769,5 +769,36 @@ public CSharpType GenResponseType(MgmtClientOperation clientOperation, bool asyn
             }
             return null;
         }
+
+        protected void WriteMethodInvocation(string methodName, List<string> paramNames)
```

Comment:
Usually if we do not need the `List` functions (like indexer, etc), we just use `IEnumerable` as type
```suggestion
        protected void WriteMethodInvocation(string methodName, IEnumerable<string> paramNames)
```